tabs/firmware_flasher.js

```diff
@@ -170,7 +170,8 @@ TABS.firmware_flasher.initialize = function (callback) {
             TABS.firmware_flasher.releases = releases;
             return;
         };
-
+        // woga65: load firmware (pre)releases from variable pitch repository
+        //$.get('https://api.github.com/repos/Woga65/inav/releases?per_page=10', function (releasesData){
```

Comment:
Likewise, I assume this is for testing. The official iNavFlight repos should be used in release trim.

Reply:
Yes, for testing. I will clean this up as soon as all probebly needed changes are discussed and applied.